wiring/src/spark_wiring_ble.cpp

```diff
@@ -1040,7 +1040,8 @@ class BleLocalDeviceImpl {
  */
 void BleCharacteristicImpl::onBleCharEvents(const hal_ble_char_evt_t *event, void* context) {
     auto impl = static_cast<BleCharacteristicImpl*>(context);
-    WiringBleLock lk;
+    // This callback won't modified any data in wiring.
```

Comment:
Can something else (e.g. application thread) modify anything accessed in this function? I don't see `peers_` being modified by non-ble event thread, but just in case asking if I've missed something.

Reply:
For sure, this callback won't modified anything in wiring **so far**. If it does, then we'll absolutely be aware of that in this callback, since we've left the comment on entering this callback.